module/VuFind/tests/integration-tests/src/VuFindTest/Mink/EmailAuthenticationTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/**
+ * Email authentication test class.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+
+namespace VuFindTest\Mink;
+
+/**
+ * Email authentication test class.
+ *
+ * Class must be final due to use of "new static()" by LiveDatabaseTrait.
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+final class EmailAuthenticationTest extends \VuFindTest\Integration\MinkTestCase
+{
+    use \VuFindTest\Feature\EmailTrait;
+    use \VuFindTest\Feature\LiveDatabaseTrait;
+
+    /**
+     * Standard setup method.
+     *
+     * @return void
+     */
+    public static function setUpBeforeClass(): void
+    {
+        static::failIfDataExists();
+    }
+
+    /**
+     * Test the email authentication process.
+     *
+     * @return void
+     */
+    public function testEmailAuthentication(): void
```

Comment:
Should this be testILSEmailAuthentication, since there's also a separate ability to use Email authentication directly which might merit a separate testStandaloneEmailAuthentication test case at some point?

Reply:
Changed, and added a test for Database+Email authentication as well.